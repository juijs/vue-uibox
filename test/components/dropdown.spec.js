import { mount } from 'vue-test-utils'
import UixDropdown from '@/components/dropdown'

describe('dropdown.js', () => {
    const wrapper = mount(UixDropdown, {
        propsData: {
            width: 200,
            items: [
                { text: "한국", index: 0 },
                { text: "미국", index: 1 },
                { text: "일본", index: 2 },
                { text: "중국", index: 3 },
                { text: "프랑스", index: 4 }
            ],
            index: 1,
            active: true
        }
    });

    it('snapshot!!', () => {
        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
})