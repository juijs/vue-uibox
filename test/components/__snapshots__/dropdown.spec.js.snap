// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dropdown.js snapshot!! 1`] = `
<div
  class="dropdown small"
  style="width: 0px; position: absolute; margin-top: 10px;"
>
  <div
    class="anchor"
  />
   
  <ul
    style="width: 200px; display: block;"
  >
    <li
      class=""
      value="0"
    >
      한국
    </li>
    <li
      class=""
      value="1"
    >
      미국
    </li>
    <li
      class=""
      value="2"
    >
      일본
    </li>
    <li
      class=""
      value="3"
    >
      중국
    </li>
    <li
      class=""
      value="4"
    >
      프랑스
    </li>
  </ul>
</div>
`;
